src/Tribe/Main.php

```diff
@@ -216,12 +236,17 @@ public static function instance() {
 		 * Initializes plugin variables and sets up WordPress hooks/actions.
 		 */
 		protected function __construct() {
-			$this->pluginPath = trailingslashit( dirname( dirname( dirname( __FILE__ ) ) ) );
-			$this->pluginDir  = trailingslashit( basename( $this->pluginPath ) );
-			$this->pluginUrl  = plugins_url( $this->pluginDir );
+			$this->pluginPath = $this->plugin_path = trailingslashit( dirname( dirname( dirname( __FILE__ ) ) ) );
```

Comment:
Instead of setting the `$this-> pluginPath` + `$this->pluginDir` + `$this-> pluginUrl` here, we could have a Magic Method `__get` set for this class to notify users that are using this class variable. What do you think @borkweb?